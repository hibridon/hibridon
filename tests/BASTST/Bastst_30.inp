    BASTST
   30                            ibasty
  F  F  F  T                     logdfl, airyfl, readpt, bastst
    5.5000    9.0000     .1400   rstart, rendld, spac
  F                              prairy
  1.160      8.00    25.0  7.00  tolai, rincr, rendai, fstfac
   1                             nerg
   442.0000
   17.13500                      xmu
   30.0000                       rcut
   0   0   1   0   0   0   1     jtot1,jtot2,jtotd,jlpar,numin,numax,nud
  48   0                         lscreen, iprint
   20    5                       nnout,niout
  00  10  20  30 40 50 60  01 11 21 31 41 51 61 02 12 22 32 42 52 62    
    0   2  -2   4  -4 
  F  F  F  F  F                  prlogd, prsmat, prt2, t2test, wrsmat
  F  F  F  F  F                  wrpart, prpart, prxsec, wrxsec, wavefl
  F  T  F  F  F                  noprin, chlist, ipos, nucros, photof
  F  F  F  F  T                  flaghf, csflag, flagsu, ihomo, twomol
  F  F                           rsflag, boundc
  4   1   0   4   1             j1max, iop, j2min, j2max, ipotsy2
 27.87700 14.51200 9.28500 1500. 0.81708446  arot, brot, crot, emax, b2rot
 nofile.dat

