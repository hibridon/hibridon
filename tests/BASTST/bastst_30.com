inp=bastst_30.inp
out=bastst_30.out
run
save=save_30.inp
