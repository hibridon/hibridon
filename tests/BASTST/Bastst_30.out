     BASTST                                      
 DUMMY POTENTIAL FOR TESTING PURPOSES            
 ==============================
     *** Parameters:
 JTOT1  =   0        JTOT2  =   0        JTOTD  =   1        JLPAR  =   0
 NERG   =   1        NUMAX  =   0        NUMIN  =   0        NUD    =   1
 LSCREEN=  48        IPRINT =   0
 FSTFAC =  7.000     RINCR  =  1.000     RCUT   =  30.00     RENDAI =  5.500    
 RENDLD =  5.500     RSTART =  5.500     SPAC   = 0.1400     TOLAI  =  1.000    
 XMU    =  17.14    
     *** ASYM-DIA system parameters:
 NTERM  =   1        NUMPOT =   0        JMAX   =   4        IOP    =   1
 J2MIN  =   0        J2MAX  =   4        IPOTSY2=   1
 AROT   =  27.88     BROT   =  14.51     CROT   =  9.285     EMAX   =  1500.    
 B2ROT  = 0.8171    
 J1/J2: 
     *** Flags:
 AIRYFL= F    BASTST= T    BATCH = F    CHLIST= F    CSFLAG= F    FLAGHF= F
 FLAGSU= F    IHOMO = F    IPOS  = F    LOGDFL= F    NOPRIN= F    NUCROS= F
 PHOTOF= F    PRAIRY= F    PRLOGD= F    PRPART= F    PRSMAT= F    PRT2  = F
 PRXSEC= F    READPT= F    RSFLAG= F    T2TEST= F    TWOMOL= T    WAVEFL= F
 WRPART= F    WRSMAT= F    WRXSEC= F    BOUNDC= F
 ** Energies:      442.000000
 NOUT: 20; J1/J2-OUT: 00  10  20  30  40  50  60  01  11  21  31  41  51  61  02  12  22  32  42  52  
 INDOUT:    0   2  -2   4  -4
 ==============================

 --------------------------------------------------------------------------
           HIBRIDON SCATTERING CODE V 5.00

      AUTHORS: M. ALEXANDER, D. MANOLOPOULOS, H.-J. WERNER, B. FOLLMEG,
               P. DAGDIGIAN
 CONTRIBUTORS: D. LEMOINE, P. VOHRALIK, G. COREY, R. JOHNSON, T. ORLIKOWSKI,
          A. BERNING, A. DEGLI-ESPOSTI, C. RIST, B. POUILLY, J. KLOS, Q. MA,
          G. VAN DER SANDEN, M. YANG, F. DE WEERD, S. GREGURICK, F. LIQUE
 --------------------------------------------------------------------------
           BUILD CONFIGURATION:
CMAKE BUILD DATE: 2023-01-16 at 19:40
CMAKE BUILD TYPE: Debug
CODE GIT REVISION: 77a341212276da93e05042bea6f46ff8e99ce91a
SYS: Darwin 22.2.0 x86_64
COMPILER: /opt/intel/oneapi/compiler/2022.2.1/mac/bin/intel64/ifort
COMPILE OPTIONS: 
 --------------------------------------------------------------------------
            CURRENT HARDWARE CONFIGURATION:
 
      Model Name: MacBook Pro
      Model Identifier: MacBookPro14,3
      Processor Name: Quad-Core Intel Core i7
      Processor Speed: 2.9 GHz
      Number of Processors: 1
      Total Number of Cores: 4
      L2 Cache (per Core): 256 KB
      L3 Cache: 8 MB
      Hyper-Threading Technology: Enabled
      Memory: 16 GB
 --------------------------------------------------------------------------

 -----------------------------------------------------------------------------
 All publications resulting from use of the integrators included in the
 Hibridon code must include the following reference:

 D. E. Manolopoulos, J. Chem. Phys. 85, 6425 (1986); M. H. Alexander and D. E.
 Manolopoulos, J. Chem. Phys. 80, 2044 (1987).

 All publications involving the determination ofphotodissociation cross sections
 must also include the following reference:

 M. H. Alexander, Comput. Phys. Commun, 75, 87 (1993).

 All publications investigating flux redistribution must include the following references:

 M. H. Alexander, J. Chem. Phys. 95, 8931 (1991); D. E. Manolopoulos and
 M. H. Alexander, J. Chem. Phys.  97, 2527 (1992).

 All publications resulting from use of the Hibridon package must include
 the following reference:

 HIBRIDON is a package of programs for the time-independent quantum treatment
 of inelastic collisions and photodissociation written by M. H. Alexander,
 D. E.  Manolopoulos, H.-J. Werner, and B. Follmeg, with contributions by
 P. F. Vohralik, D. Lemoine, G. Corey, B. Johnson, T. Orlikowski, A. Berning,
 A. Degli-Esposti, C. Rist, P. Dagdigian, B. Pouilly, G. van der Sanden, M. Yang, F. de Weerd, S. Gregurick, J. Klos, and F. Lique
 -----------------------------------------------------------------------------

 ** LABEL:         BASTST                                      
 ** POT NAME:  DUMMY POTENTIAL FOR TESTING PURPOSES            
 ** DATE:      Jan 16 19:53:04 2023

 *** WARNING:  NAIRY .NE. 1 BUT AIRYFL .EQ. .FALSE.
 ** NMAX= 500  NAIRY= 500

 **  CC ASYMMETRIC TOP + LINEAR MOLECULE **
     RMU=  17.1350  AROT= 27.877  BROT= 14.512  CROT=  9.285  B2ROT=  0.817
     E= 442.00  JTOT=   0   JLPAR= 1

  ASYMMETRIC TOP + LINEAR MOLECULE LEVELS
 -----------------------------------------------------------------
   #  J1  IS  KP  KO  J2  Eint(cm-1)  Coeffs
 -----------------------------------------------------------------

  CC CHANNEL BASIS
 --------------------------------------------------
   #  J1  IS  J2  J12  L     EINT(CM-1)
 --------------------------------------------------
   1   0   0   0   0   0       0.000
   2   0   0   1   1   1       1.634
   3   0   0   2   2   2       4.903
   4   0   0   3   3   3       9.805
   5   0   0   4   4   4      16.342
   6   1  -1   0   1   1      23.797
   7   1  -1   1   0   0      25.431
   8   1  -1   1   2   2      25.431
   9   1  -1   2   1   1      28.700
  10   1  -1   2   3   3      28.700
  11   1  -1   3   2   2      33.602
  12   1  -1   3   4   4      33.602
  13   1   0   0   1   1      37.162
  14   1   0   1   0   0      38.796
  15   1   0   1   2   2      38.796
  16   1  -1   4   3   3      40.139
  17   1  -1   4   5   5      40.139
  18   1   0   2   1   1      42.065
  19   1   0   2   3   3      42.065
  20   1   1   1   1   1      44.023
  21   1   0   3   2   2      46.967
  22   1   0   3   4   4      46.967
  23   1   1   2   2   2      47.292
  24   1   1   3   3   3      52.194
  25   1   0   4   3   3      53.504
  26   1   0   4   5   5      53.504
  27   1   1   4   4   4      58.731
  28   2   0   0   2   2      70.133
  29   2   0   1   1   1      71.767
  30   2   0   1   3   3      71.767
  31   2   0   2   0   0      75.036
  32   2   0   2   2   2      75.036
  33   2   0   2   4   4      75.036
  34   2  -1   0   2   2      79.529
  35   2   0   3   1   1      79.938
  36   2   0   3   3   3      79.938
  37   2   0   3   5   5      79.938
  38   2  -1   1   1   1      81.163
  39   2  -1   1   3   3      81.163
  40   2  -1   2   0   0      84.432
  41   2  -1   2   2   2      84.432
  42   2  -1   2   4   4      84.432
  43   2   0   4   2   2      86.475
  44   2   0   4   4   4      86.475
  45   2   0   4   6   6      86.475
  46   2  -1   3   1   1      89.334
  47   2  -1   3   3   3      89.334
  48   2  -1   3   5   5      89.334
  49   2  -1   4   2   2      95.871
  50   2  -1   4   4   4      95.871
  51   2  -1   4   6   6      95.871
  52   2  -2   1   2   2      96.844
  53   2  -2   2   1   1     100.113
  54   2  -2   2   3   3     100.113
  55   2  -2   3   2   2     105.015
  56   2  -2   3   4   4     105.015
  57   2  -2   4   3   3     111.552
  58   2  -2   4   5   5     111.552
  59   2   2   0   2   2     136.563
  60   3  -1   0   3   3     136.909
  61   2   1   1   2   2     136.939
  62   2   2   1   1   1     138.197
  63   2   2   1   3   3     138.197
  64   3  -1   1   2   2     138.544
  65   3  -1   1   4   4     138.544
  66   2   1   2   1   1     140.208
  67   2   1   2   3   3     140.208
  68   2   2   2   0   0     141.465
  69   2   2   2   2   2     141.465
  70   2   2   2   4   4     141.465
  71   3  -1   2   1   1     141.812
  72   3  -1   2   3   3     141.812
  73   3  -1   2   5   5     141.812
  74   3   0   0   3   3     142.369
  75   3   0   1   2   2     144.003
  76   3   0   1   4   4     144.003
  77   2   1   3   2   2     145.110
  78   2   1   3   4   4     145.110
  79   2   2   3   1   1     146.368
  80   2   2   3   3   3     146.368
  81   2   2   3   5   5     146.368
  82   3  -1   3   0   0     146.714
  83   3  -1   3   2   2     146.714
  84   3  -1   3   4   4     146.714
  85   3  -1   3   6   6     146.714
  86   3   0   2   1   1     147.272
  87   3   0   2   3   3     147.272
  88   3   0   2   5   5     147.272
  89   2   1   4   3   3     151.647
  90   2   1   4   5   5     151.647
  91   3   0   3   0   0     152.174
  92   3   0   3   2   2     152.174
  93   3   0   3   4   4     152.174
  94   3   0   3   6   6     152.174
  95   2   2   4   2   2     152.904
  96   2   2   4   4   4     152.904
  97   2   2   4   6   6     152.904
  98   3  -1   4   1   1     153.251
  99   3  -1   4   3   3     153.251
 100   3  -1   4   5   5     153.251
 101   3  -1   4   7   7     153.251
 102   3   0   4   1   1     158.711
 103   3   0   4   3   3     158.711
 104   3   0   4   5   5     158.711
 105   3   0   4   7   7     158.711
 106   3   1   1   3   3     175.169
 107   3   1   2   2   2     178.438
 108   3   1   2   4   4     178.438
 109   3   1   3   1   1     183.340
 110   3   1   3   3   3     183.340
 111   3   1   3   5   5     183.340
 112   3   1   4   2   2     189.877
 113   3   1   4   4   4     189.877
 114   3   1   4   6   6     189.877
 115   3  -2   1   3   3     208.330
 116   3  -2   2   2   2     211.599
 117   3  -2   2   4   4     211.599
 118   3  -3   0   3   3     212.569
 119   3  -3   1   2   2     214.203
 120   3  -3   1   4   4     214.203
 121   3  -2   3   1   1     216.501
 122   3  -2   3   3   3     216.501
 123   3  -2   3   5   5     216.501
 124   3  -3   2   1   1     217.471
 125   3  -3   2   3   3     217.471
 126   3  -3   2   5   5     217.471
 127   4   0   0   4   4     222.369
 128   3  -3   3   0   0     222.374
 129   3  -3   3   2   2     222.374
 130   3  -3   3   4   4     222.374
 131   3  -3   3   6   6     222.374
 132   3  -2   4   2   2     223.038
 133   3  -2   4   4   4     223.038
 134   3  -2   4   6   6     223.038
 135   4   0   1   3   3     224.004
 136   4   0   1   5   5     224.004
 137   4  -1   0   4   4     225.068
 138   4  -1   1   3   3     226.702
 139   4  -1   1   5   5     226.702
 140   4   0   2   2   2     227.272
 141   4   0   2   4   4     227.272
 142   4   0   2   6   6     227.272
 143   3  -3   4   1   1     228.910
 144   3  -3   4   3   3     228.910
 145   3  -3   4   5   5     228.910
 146   3  -3   4   7   7     228.910
 147   4  -1   2   2   2     229.970
 148   4  -1   2   4   4     229.970
 149   4  -1   2   6   6     229.970
 150   4   0   3   1   1     232.174
 151   4   0   3   3   3     232.174
 152   4   0   3   5   5     232.174
 153   4   0   3   7   7     232.174
 154   4  -1   3   1   1     234.873
 155   4  -1   3   3   3     234.873
 156   4  -1   3   5   5     234.873
 157   4  -1   3   7   7     234.873
 158   4   0   4   0   0     238.711
 159   4   0   4   2   2     238.711
 160   4   0   4   4   4     238.711
 161   4   0   4   6   6     238.711
 162   4   0   4   8   8     238.711
 163   4  -1   4   0   0     241.409
 164   4  -1   4   2   2     241.409
 165   4  -1   4   4   4     241.409
 166   4  -1   4   6   6     241.409
 167   4  -1   4   8   8     241.409
 168   4  -2   1   4   4     277.649
 169   4  -2   2   3   3     280.917
 170   4  -2   2   5   5     280.917
 171   4  -2   3   2   2     285.820
 172   4  -2   3   4   4     285.820
 173   4  -2   3   6   6     285.820
 174   3   2   0   3   3     287.299
 175   3   2   1   2   2     288.933
 176   3   2   1   4   4     288.933
 177   3   3   1   3   3     289.129
 178   3   2   2   1   1     292.201
 179   3   2   2   3   3     292.201
 180   3   2   2   5   5     292.201
 181   4  -2   4   1   1     292.356
 182   4  -2   4   3   3     292.356
 183   4  -2   4   5   5     292.356
 184   4  -2   4   7   7     292.356
 185   3   3   2   2   2     292.397
 186   3   3   2   4   4     292.397
 187   3   2   3   0   0     297.104
 188   3   2   3   2   2     297.104
 189   3   2   3   4   4     297.104
 190   3   2   3   6   6     297.104
 191   3   3   3   1   1     297.300
 192   3   3   3   3   3     297.300
 193   3   3   3   5   5     297.300
 194   4   1   1   4   4     302.526
 195   3   2   4   1   1     303.641
 196   3   2   4   3   3     303.641
 197   3   2   4   5   5     303.641
 198   3   2   4   7   7     303.641
 199   3   3   4   2   2     303.836
 200   3   3   4   4   4     303.836
 201   3   3   4   6   6     303.836
 202   4   1   2   3   3     305.794
 203   4   1   2   5   5     305.794
 204   4   1   3   2   2     310.697
 205   4   1   3   4   4     310.697
 206   4   1   3   6   6     310.697
 207   4   2   0   4   4     316.467
 208   4   1   4   1   1     317.233
 209   4   1   4   3   3     317.233
 210   4   1   4   5   5     317.233
 211   4   1   4   7   7     317.233
 212   4   2   1   3   3     318.101
 213   4   2   1   5   5     318.101
 214   4   2   2   2   2     321.370
 215   4   2   2   4   4     321.370
 216   4   2   2   6   6     321.370
 217   4   2   3   1   1     326.272
 218   4   2   3   3   3     326.272
 219   4   2   3   5   5     326.272
 220   4   2   3   7   7     326.272
 221   4   2   4   0   0     332.809
 222   4   2   4   2   2     332.809
 223   4   2   4   4   4     332.809
 224   4   2   4   6   6     332.809
 225   4   2   4   8   8     332.809
 226   4  -3   0   4   4     384.522
 227   4  -3   1   3   3     386.157
 228   4  -3   1   5   5     386.157
 229   4  -4   1   4   4     387.479
 230   4  -3   2   2   2     389.425
 231   4  -3   2   4   4     389.425
 232   4  -3   2   6   6     389.425
 233   4  -4   2   3   3     390.748
 234   4  -4   2   5   5     390.748
 235   4  -3   3   1   1     394.327
 236   4  -3   3   3   3     394.327
 237   4  -3   3   5   5     394.327
 238   4  -3   3   7   7     394.327
 239   4  -4   3   2   2     395.650
 240   4  -4   3   4   4     395.650
 241   4  -4   3   6   6     395.650
 242   4  -3   4   0   0     400.864
 243   4  -3   4   2   2     400.864
 244   4  -3   4   4   4     400.864
 245   4  -3   4   6   6     400.864
 246   4  -3   4   8   8     400.864
 247   4  -4   4   1   1     402.187
 248   4  -4   4   3   3     402.187
 249   4  -4   4   5   5     402.187
 250   4  -4   4   7   7     402.187
 251   4   4   0   4   4     494.643
 252   4   3   1   4   4     496.252
 253   4   4   1   3   3     496.278
 254   4   4   1   5   5     496.278
 255   4   3   2   3   3     499.521
 256   4   3   2   5   5     499.521
 257   4   4   2   2   2     499.546
 258   4   4   2   4   4     499.546
 259   4   4   2   6   6     499.546
 260   4   3   3   2   2     504.423
 261   4   3   3   4   4     504.423
 262   4   3   3   6   6     504.423
 263   4   4   3   1   1     504.448
 264   4   4   3   3   3     504.448
 265   4   4   3   5   5     504.448
 266   4   4   3   7   7     504.448
 267   4   3   4   1   1     510.960
 268   4   3   4   3   3     510.960
 269   4   3   4   5   5     510.960
 270   4   3   4   7   7     510.960
 271   4   4   4   0   0     510.985
 272   4   4   4   2   2     510.985
 273   4   4   4   4   4     510.985
 274   4   4   4   6   6     510.985
 275   4   4   4   8   8     510.985

 ILAM  L1   M1   L2  LTOT    ICOL  IROW       I       IV2       VEE
 *** TOTAL NUMBER OF NONZERO V2 MATRIX ELEMENTS IS         0
 ** BASTST=.TRUE.; TEST OF SUBROUTINE BASIS
 **** END OF CALCULATION ****
      MAXIMUM NUMBER OF CHANNELS USED WAS:     0
      TIMING:  ELAPSED 00:00:00 / CPU 00:00:00 / MP RATIO    6.4 %
      CURRENT DATE:  Jan 16 19:53:04 2023
 ===============================================================================
